script/upgrades/MU03/MU03.sol

```diff
@@ -0,0 +1,506 @@
+            )
+          )
+        );
+      }
+    }
+
+    // Enable Value Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isValueDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              valueDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Set BreakerBox address in SortedOracles
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        contracts.celoRegistry("SortedOracles"),
+        abi.encodeWithSelector(SortedOracles(0).setBreakerBox.selector, breakerBox)
+      )
+    );
+  }
+
+  function proposal_configureMedianDeltaBreaker() public {
+    // Set the cooldown times
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        medianDeltaBreaker,
+        abi.encodeWithSelector(
+          MedianDeltaBreaker(0).setCooldownTime.selector,
+          Arrays.addresses(cUSD, cEUR, cBRL, cEURUSDCConfig.referenceRateFeedID, cBRLUSDCConfig.referenceRateFeedID),
+          Arrays.uints(
+            cUSDCeloConfig.medianDeltaBreakerCooldown,
+            cEURCeloConfig.medianDeltaBreakerCooldown,
+            cBRLCeloConfig.medianDeltaBreakerCooldown,
+            cEURUSDCConfig.medianDeltaBreakerCooldown,
+            cBRLUSDCConfig.medianDeltaBreakerCooldown
+          )
+        )
+      )
+    );
+
+    // Set the rate change thresholds
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        medianDeltaBreaker,
+        abi.encodeWithSelector(
+          MedianDeltaBreaker(0).setRateChangeThresholds.selector,
+          Arrays.addresses(cUSD, cEUR, cBRL, cEURUSDCConfig.referenceRateFeedID, cBRLUSDCConfig.referenceRateFeedID),
+          Arrays.uints(
+            cUSDCeloConfig.medianDeltaBreakerThreshold.unwrap(),
+            cEURCeloConfig.medianDeltaBreakerThreshold.unwrap(),
+            cBRLCeloConfig.medianDeltaBreakerThreshold.unwrap(),
+            cEURUSDCConfig.medianDeltaBreakerThreshold.unwrap(),
+            cBRLUSDCConfig.medianDeltaBreakerThreshold.unwrap()
+          )
+        )
+      )
+    );
+
+    // Set smoothing factor for rate feeds
+    for (uint i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isMedianDeltaBreakerEnabled && poolConfigs[i].smoothingFactor != 0)
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            medianDeltaBreaker,
+            abi.encodeWithSelector(
+              MedianDeltaBreaker(0).setSmoothingFactor.selector,
+              poolConfigs[i].referenceRateFeedID,
+              poolConfigs[i].smoothingFactor
+            )
+          )
+        );
+    }
+  }
+
+  function proposal_configureValueDeltaBreaker() public {
```

Comment:
could have done the same here as in the `proposal_configureMedianDeltaBreaker` and iterate over all the poolConfigs and check for `isValueDeltaBreakerEnabled`